config/runtime.exs

```diff
@@ -21,20 +21,9 @@ if System.get_env("PHX_SERVER") do
 end
 
 if config_env() == :prod do
-  database_url =
-    System.get_env("DATABASE_URL") ||
-      raise """
-      environment variable DATABASE_URL is missing.
-      For example: ecto://USER:PASS@HOST/DATABASE
-      """
-
-  maybe_ipv6 = if System.get_env("ECTO_IPV6") in ~w(true 1), do: [:inet6], else: []
-
   config :phil, Phil.Repo,
-    # ssl: true,
-    url: database_url,
-    pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
-    socket_options: maybe_ipv6
+    ssl: true,
+    configure: {MyCharlie.Repo, :configure_production_connection, []}
```

Comment:
suggestion: maybe this?
```suggestion
    configure: {Phil.Repo, :configure_production_connection, []}
```

Reply:
This was the issue. Thanks for your sharp eyes!